kf_hungarian_node:
  ros__parameters:
    death_threshold: 3 # maximum mising frames before deleting an obstacle
    # Kamlan filter related
    top_down: False # whether use top-down (x, y) view or 3D (x, y, z)
    measurementNoiseCov: [1., 1., 1.] # if it's use top-down view, set the 3rd number to 0
    errorCovPost: [1., 1., 1., 10., 10., 10.] # if use top-down view, set the 3rd and 6th to 0
    a_noise: [2., 2., 0.5] # acceration noise, if use top-down view, the 3rd number should be 0
    # obstacle filter
    vel_filter: 0.1 # only keep obstacles with velocities greater than this value
    cost_filter: 1.0 # filter Hungarian assignment with cost greater than threshold
    